src/components/Provider/index.tsx

```diff
@@ -28,6 +29,12 @@ const Provider = ({ theme, ...props }: ThemedProvider) => {
 	const providerTheme = merge(defaultTheme, theme);
 	return (
 		<BreakpointProvider breakpoints={providerTheme.breakpoints}>
+			<Helmet>
+				<link
+					href="https://fonts.googleapis.com/css?family=Nunito|Ubuntu+Mono:400,700|Source+Sans+Pro"
```

Comment:
```suggestion
					href="https://fonts.googleapis.com/css?family=Nunito|Ubuntu+Mono:400,700|Source+Sans+Pro&display=fallback"
```

Reply:
Also, let's see what weights we use from Nunito and Source Sans Pro and only import those. This is something we need to chat with Alexandra, can you please chat with her? BTW the idea is to also replace nunito with source sans pro, but I'd be happy if we do the titles first.